lib/watcher-middleware.js

```diff
@@ -34,6 +36,23 @@ module.exports = function watcherMiddleware(watcher, options) {
 
       next();
     }, (buildError) => {
+      if (request.url === '/ember_open_editor_for_template_compilation_error') {
+        let errorLocation = buildError.broccoliPayload.error.location || {};
+        let fileName = errorLocation.file;
+        if (fileName) {
```

Comment:
the broccoliPayload object does not have a file name...will explore adding it as the stack frame has it.